src/org/apache/xerces/impl/XML11NSDocumentScannerImpl.java

```diff
@@ -130,6 +132,20 @@ protected boolean scanStartElement() throws IOException, XNIException {
 
         // Note: namespace processing is on by default
         fEntityScanner.scanQName(fElementQName);
+
```

Comment:
There is almost the same code in this class and in the XMLDocumentFragmentScanner. 
I hate duplicating code.

Is it needed in both places if it is to work with XML 1.0 and XML 1.1 ? Is there a sensible place to factor it out?